index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Mobile First Project</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"/>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+
+    <header class="main-header">
+
+            <nav class="header">
+
+                <div class="name"><a href="#" style="color: white; text-decoration: none;">Paul Orife</a></div>
+                <div class="menu"><i class="fas fa-bars"></i></div>
```

Comment:
- [ ] Kindly update your layout so the hamburger is shown on screen size 375px.
<img src="https://user-images.githubusercontent.com/17243325/204979884-d3aef81b-3746-44cc-a0a5-73a8ada0282e.png" width="300">